packages/doc/src/components/Clipboard/index.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-undef */
+import React, { useState } from 'react';
+import { Button, Box } from '@gympass/yoga';
+import { string } from 'prop-types';
+
+import { CheckedFull, Copy } from '@gympass/yoga-icons/src';
+
+const Clipboard = ({ copyText }) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  async function CopyTextToClipboard(text) {
+    const textToCopy = await navigator.clipboard.writeText(text);
+
+    return textToCopy;
+  }
+
+  const handleCopyClick = () => {
+    CopyTextToClipboard(copyText)
+      .then(() => {
+        setIsCopied(true);
+        setTimeout(() => {
+          setIsCopied(false);
+        }, 1500);
+      })
+      .catch(error => {
+        throw new Error(error);
+      });
```

Comment:
Maybe we could write this using `async`/`await`. I also think that we don't need the catch block if we are just throwing the error again.

```suggestion
    await copyTextToClipboard(copyText);
    setIsCopied(true);
    setTimeout(() => {
      setIsCopied(false);
    }, 1500);
```

Reply:
make sense! Thanks!